app/helpers/js_helper.rb

```diff
@@ -9,51 +9,44 @@ def set_spinner_off
     'miqSparkleOff();'
   end
 
-  # safe variant of j/escape_javascript that calls .to_s to work with non-string values
-  def j_str(value)
-    j(value.to_s)
-  end
-
   def javascript_focus(element)
-    "$('##{j_str(element)}').focus();".html_safe
+    "$('##{j(element)}').focus();".html_safe
   end
 
   def javascript_disable_field(element)
-    "$('##{j_str(element)}').prop('disabled', true);".html_safe
+    "$('##{j(element)}').prop('disabled', true);".html_safe
   end
 
   def javascript_enable_field(element)
-    "$('##{j_str(element)}').prop('disabled', false);".html_safe
+    "$('##{j(element)}').prop('disabled', false);".html_safe
   end
 
   def javascript_show(element)
-    "$('##{j_str(element)}').show();".html_safe
+    "$('##{j(element)}').show();".html_safe
   end
 
   def javascript_hide(element)
-    "$('##{j_str(element)}').hide();".html_safe
+    "$('##{j(element)}').hide();".html_safe
   end
 
   def javascript_show_if_exists(element)
-    "if (miqDomElementExists('#{j_str(element)}')) #{javascript_show(element)}".html_safe
+    "if (miqDomElementExists('#{j(element)}')) #{javascript_show(element)}".html_safe
   end
 
   def javascript_hide_if_exists(element)
-    "if (miqDomElementExists('#{j_str(element)}')) #{javascript_hide(element)}".html_safe
+    "if (miqDomElementExists('#{j(element)}')) #{javascript_hide(element)}".html_safe
   end
 
   def partial_replace(from, partial, locals)
-    "$(\"##{h(from)}\").replaceWith(\"#{escape_javascript(render(:partial => partial, :locals => locals))}\");".html_safe
+    "$(\"##{h(from)}\").replaceWith(\"#{j(render(:partial => partial, :locals => locals))}\");".html_safe
```

Comment:
escape_javascript and j are aliases, so this just aligns with `j` as this was the only outlier. Even so, I'm deleting this in #9436